sys/fido2/ctap/Kconfig

```diff
@@ -84,12 +84,22 @@ config FIDO2_CTAP_UP_BUTTON_FLANK_RISING
 
 endchoice
 
-config FIDO2_CTAP_FLASH_START_PAGE
-    int "First flash page to store data in"
-    default -1
+config FIDO2_CTAP_NUM_FLASHPAGES
+    int "Amount of flashpages to use"
+    range 2 256
+    default 4
     help
-        Configuring this incorrectly can lead to firmware corruption so make sure
-        the flash page is located after the firmware.
+        Configure how many flashpages are used to store FIDO2 CTAP data.
+
+        To save a credential in flash memory, roughly 156 bytes are needed. This
+        number might change slightly depending on the flash block size.
+        Therefore, if one wants to e.g. save 40 credentials and the flashpage
+        size is 4096 bytes roughly 156 * 40 / 4096 (2) flashpages are needed.
+        To save authenticator state data one additional flashpage is needed.
+        So in total one has to configure 3 to save 40 credentials.
+
+        Configuring the amount too high can lead to loss of FIDO2 data as a
```

Comment:
Why not check against `flashpage_last_free()`?

Could probably not be caught at Kconfig time, and maybe not even at build time, but a runtime error ("Sorry can't use my flash" or "You only get this much flash rather than what you configured; sorry"), but all that would still be better than silent failure.

Reply:
Just saw that this is already being discussed somewhere else in the code; please mark as resolved once that's done.